packages/salesforcedx-vscode-replay-debugger/package.json

```diff
@@ -72,6 +73,15 @@
         "language": "apex"
       }
     ],
+    "views": {
+      "debug": [
+        {
+          "id": "sfdx.force.view.checkpoint",
+          "name": "%view_checkpoints%",
+          "when": "sfdx:project_opened"
```

Comment:
Checkpoints panel is only visible when salesforcedx-vscode-core extension sets this context.